src/app/scheduling/datetimeeditor/datetimeeditor-sample-1/datetimeeditor-sample-1.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from "@angular/core";
+import {
+    IgxDateTimeEditorDirective,
+    IgxDateTimeEditorEventArgs,
+    IgxInputGroupModule
+} from "igniteui-angular";
```

Comment:
Remove all of the unused imports from datetimeeditor-sample-1.component.ts :
import {
    IgxDateTimeEditorDirective,
    IgxDateTimeEditorEventArgs,
    IgxInputGroupModule
} from "igniteui-angular";